azure-pipelines-daily.yaml

```diff
@@ -84,6 +84,16 @@ parameters:
           resource_group: cft-perftest-00-rg
           name: cft-perftest-00-aks
 
+      - deployment: test_dt_availability_dashboards_yaml
+        environment: test
```

Comment:
Would it cause issues to use `perftest` here? I know in SDS it's test, but likely this would create whole new dashboard which I'm not sure if you want / we should consider for SDS

Let's see either way 😁

Reply:
sure make sense - I have changed it now